packages/openneuro-app/src/scripts/datalad/fragments/freshdesk-widget.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import { useCookies } from 'react-cookie'
-import { getProfile } from '../../refactor_2021/authentication/profile'
+import { getProfile } from '../../authentication/profile'
```

Comment:
Should this file have gone into dataset/fragments instead of datalad/fragments?

Reply:
Yeah, it probably shouldn't be in there. This component is only used in the error boundary component, so I think it should be next to that.